standalone/runtime/src/lib.rs

```diff
@@ -1399,6 +1399,7 @@ parameter_types! {
     pub const MaxPriorities: u32 = 3;
     pub const CollectionSymbolLimit: u32 = 100;
     pub const MaxResourcesOnMint: u32 = 100;
+    pub const WPhaMinBalance: Balance = CENTS;
```

Comment:
Is it too large? I think as long as it's larger than the ED, it will be fine.

However, no need to change it now. We will go with the current code, and change it later when we have time.

Reply:
It is set when asset created, we set it to a cent and if lower than a cent it will throw errors.